[2023-07-12 17:04:32,107] WARNING cadastra_aluno() L48   Erro ao cadastrar aluno 'aluno', Aluno já cadastrado na base :/ - call_trace=C:\Users\gipri\PycharmProjects\university\app.py L48  
[2023-07-12 17:04:32,116] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:04:32] "[31m[1mPOST /aluno HTTP/1.1[0m" 409 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:04:59,361] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:04:59] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:05:39,085] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:05:39] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:05:39,087] WARNING del_aluno() L105  Erro ao excluir aluno #'Luciana da Silva Santos', Aluno não encontrado na base :/ - call_trace=C:\Users\gipri\PycharmProjects\university\app.py L105 
[2023-07-12 17:05:39,087] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:05:39] "[33mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 404 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:05:43,900] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:05:43] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:06:09,124] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:06:09] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:06:09,133] WARNING del_aluno() L105  Erro ao excluir aluno #'Luciana da Silva Santos', Aluno não encontrado na base :/ - call_trace=C:\Users\gipri\PycharmProjects\university\app.py L105 
[2023-07-12 17:06:09,135] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:06:09] "[33mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 404 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:06:14,170] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:06:14] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:06:15,986] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:06:15] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:08:05,817] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:08:05] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:08:05,826] WARNING del_aluno() L105  Erro ao excluir aluno #'Luciana da Silva Santos', Aluno não encontrado na base :/ - call_trace=C:\Users\gipri\PycharmProjects\university\app.py L105 
[2023-07-12 17:08:05,828] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:08:05] "[33mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 404 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 17:08:11,699] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 17:08:11] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:23:50,501] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.28.49:5000 (Press CTRL+C to quit) - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:14,680] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:14] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:27,163] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:27] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:27,168] ERROR log_exception() L1449 Exception on /aluno [DELETE] - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_openapi3\scaffold.py", line 355, in wrapper
    resp = _do_wrapper(
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "C:\Users\gipri\PycharmProjects\university\app.py", line 94, in del_aluno
    aluno_nome = unquote(unquote(query.nome))
AttributeError: 'AlunoGetSchema' object has no attribute 'nome'
[2023-07-12 18:24:27,224] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:27] "[35m[1mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 500 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:33,108] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:33] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:59,058] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:59] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:24:59,062] ERROR log_exception() L1449 Exception on /aluno [DELETE] - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_openapi3\scaffold.py", line 355, in wrapper
    resp = _do_wrapper(
  File "C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "C:\Users\gipri\PycharmProjects\university\app.py", line 94, in del_aluno
    aluno_nome = unquote(unquote(query.nome))
AttributeError: 'AlunoGetSchema' object has no attribute 'nome'
[2023-07-12 18:24:59,068] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:24:59] "[35m[1mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 500 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:30:08,008] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.28.49:5000 (Press CTRL+C to quit) - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:30:35,440] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:30:35] "GET /alunos HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:30:57,711] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:30:57] "OPTIONS /aluno?cpf=aluno%20aluno HTTP/1.1" 200 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
[2023-07-12 18:30:57,717] WARNING del_aluno() L106  Erro ao excluir aluno #'Luciana da Silva Santos', Aluno não encontrado na base :/ - call_trace=C:\Users\gipri\PycharmProjects\university\app.py L106 
[2023-07-12 18:30:57,718] INFO _log() L224  127.0.0.1 - - [12/Jul/2023 18:30:57] "[33mDELETE /aluno?cpf=aluno%20aluno HTTP/1.1[0m" 404 - - call_trace=C:\Users\gipri\PycharmProjects\university\venv\lib\site-packages\werkzeug\_internal.py L224 
